src/main/R/HHC_plot.R

```diff
@@ -0,0 +1,55 @@
+library(ggplot2)
+
+# load HHC and CGD sets
+hhc <- read.table("/Users/joeri/github/rvcf/src/test/resources/HomHetCounts_r1.0.tsv", sep="\t", header=T)
+cgd <- read.table("/Users/joeri/github/rvcf/src/test/resources/GenesInheritance31aug2016.tsv", sep="\t", header=T)
+
```

Comment:
hardcoded paths, not sure how to fix in R, can those be relative to the project?